src/components/BulkEnrollmentPage/index.jsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import Hero from '../Hero';
 
 import CourseSearch from './CourseSearch';
 
 function BulkEnrollmentPage({ enterpriseId }) {
   return (
-    <>
-      <CourseSearch
-        enterpriseId={enterpriseId}
-      />
-    </>
+    <div className="container-fluid">
```

Comment:
Naming wise I see `Catalog management`, then we have 'bulk enrollment' within that, just some food for naming thoughts.. nothing for this PR

Reply:
Yeah, I think it'll be best if we rename a bunch of things, maybe open a ticket on the epic?

Reply:
yeah we can talk about it more? Not sure what Joe's vision on catalog management vs bulk enrollment is, are they one and the same? I had not even seem catalog management before hence was querying around about this. Best to ask Joe I think

Reply:
I will quickly ask on slack to get more clarity but yeah no action needed for this PR

Reply:
All of the Miro mocks showed Catalog management so on my PR I was the one to update this naming. Probably best to just reach out for final say from Product